.github/workflows/ci.yml

```
@@ -35,14 +42,16 @@ jobs:
         if: "${{ env.DOCKER_USERNAME != '' }}"
         uses: docker/metadata-action@v4
         with:
-          images: ${{ secrets.DOCKER_USERNAME }}/jellyplex-watched # list of Docker images to use as base name for tags
+          images: |
+            ${{ secrets.DOCKER_USERNAME }}/jellyplex-watched
+            ghcr.io/${{ secrets.DOCKER_USERNAME }}/jellyplex-watched
           tags: |
-            type=raw,value=latest,enable={{is_default_branch}}
-            type=ref,event=branch
-            type=ref,event=pr
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=sha
+            type=raw,value=${{ matrix.variant }},enable={{is_default_branch}}

```

I think something like this might work
```suggestion
            type=raw,value=latest,enable={{ is_default_branch && matrix.variant == 'alpine' }} # Latest tag
            type=raw,value=latest,suffix=-${{ matrix.variant }},enable={{is_default_branch}} # Latest-variant tag

```

---

I'll edit it as you put it, but just to say I've rarely seen `variant-latest` format, it's usually just been `variant`

---

For some reason it doesn't work if I use is_default_branch so I used an alternative to check if it's the default branch